src/components/ui/navbar.tsx

```
@@ -1,61 +1,209 @@
-import { Menu, Settings, User } from "lucide-react";
+import { Bell, ChevronDown, LogOut, Menu, Settings, User, Wallet } from "lucide-react";
```

ESLint

[error] 'Bell' is defined but never used.

(@typescript-eslint/no-unused-vars)